src/current/v23.2/log-sql-statistics-to-datadog.md

```diff
@@ -60,6 +60,17 @@ sinks:
       channels: [] # set to empty square brackets
 ~~~
 
+{{site.data.alerts.callout_success}}
+If you prefer to keep the `DD-API-KEY` in a file other than the `logs.yaml`, replace the `headers` parameter with the [`file-based-headers` parameter]({% link {{ page.version.version }}/configure-logs.md %}#file-based-headers):
+
+{% include_cached copy-clipboard.html %}
+~~~ yaml
+      file-based-headers: {DD-API-KEY: "{path/to/file}"} # replace with path of file containing DATADOG API key
```

Comment:
Same comment as above. I think the brackets make the example confusing, and recommend going with something like:

```
file-based-headers: DD-API-KEY: "/some/path/to/file"
```

Reply:
I removed curly brackets around variable placeholder. I kept curly brackets for YAML to define mapping.